src/Internal/Wallet/Cip30.purs

```diff
@@ -92,16 +93,12 @@ type Cip30Wallet =
   }
 
 mkCip30WalletAff
-  :: String
-  -- ^ Name of the wallet for error messages
-  -> Effect (Promise Cip30Connection)
+  :: Effect (Promise Cip30Connection)
   -- ^ A function to get wallet connection
   -> Aff Cip30Wallet
-mkCip30WalletAff walletName enableWallet = do
+mkCip30WalletAff enableWallet = do
   wallet <- toAffE enableWallet
   -- Ensure the Nami wallet has collateral set up
-  whenM (isNothing <$> getCollateral wallet) do
-    liftEffect $ throw $ walletName <> " wallet missing collateral"
```

Comment:
The `getCollateral` function from the `Cip30Wallet` class already returns a `Maybe (Array ...)`, which means that its users must handle the case where `getCollateral` fails (i.e. no collateral is available). Therefore, the sanity check in this case is unnecessary and can be removed. If it is needed, it should be moved to a different location, as Yoroi uses a different collateral function (`getCollateralYoroi`) and there is no way (other than case-matching on `walletName :: String`) to determine which wallet is being created at this stage.

Reply:
Then comment should be killed to, no?